### ISSUE TITLE
the specific parameters of model  fine-tuned on English data in task "Cross-lingual zero-shot transfer for XNLI"

### ISSUE THREAD

**Issue by 27182812** [google-research/multilingual-t5#95]
Hello, could you tell me the specific parameters of model  fine-tuned on English data in task "Cross-lingual zero-shot transfer for XNLI", such as learning rate, epochs, batchsize,  GPU, etc.
